src/utils.cpp

```diff
@@ -0,0 +1,74 @@
+#include "Bezier/utils.h"
+
+#include <numeric>
+
+using namespace Bezier;
+using namespace Bezier::Utils;
+
+PointVector Bezier::Utils::_polylineSimplify(const PointVector& polyline, unsigned int N)
```

Comment:
yes!
it is already fixed in later branches that are coming after this PR.
If I can, I would like to avoid conflicts for such trivial stuff + there are two more such occurences in bezier.cpp (joinCurves and fromPolyline function declaration)

I pinky swear I'll fix it in subsequent PRs to devel :)

Reply:
Well, if you pinky swear..